api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/Parameter.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.h2o.sparkling.api.generation.common
+
+case class Parameter(
+  name: String,
```

Comment:
Do I understand correctly that `name` is SW name and `originalName` is H2O name?

Great idea! Right now we don't store the mapping in the code but could be used in few places -> mainly Grid for the SW -> h2o param mapping

Reply:
I will rename it to be more descriptive, thanks!